### ISSUE TITLE
Unwanted alt text for design elements

### ISSUE THREAD

**Issue by bravegnu** [pythonindia/inpycon2019#77]
The idea of alt is to help people with text based client and screen readers, to understand better what is displayed on the website.

So alt, for content photos like keynote speaker, venue, etc. should have a text that best describes what is being shown, so that text client and screen readers can show them in-place of the image.

But for images that are used purely for decorative purposes, these add unwanted noise to the screen readers, and text client. The screenshot below shows the unnecessary "Cloud", "Wind", "Sun", "train", "railway", "Bracket", etc. that add to the noise.

![text-client](https://user-images.githubusercontent.com/1171852/58003212-f22f2000-7afd-11e9-9022-b49ee287aea2.png)

**Comment by bravegnu** [comment]
For decorative images the alt should be `alt=""`

**Comment by astronomersiva** [comment]
Fixed.
